doc/man3/RAND_DRBG_reseed.pod

```diff
@@ -13,7 +13,8 @@ RAND_DRBG_set_reseed_defaults
  #include <openssl/rand_drbg.h>
 
  int RAND_DRBG_reseed(RAND_DRBG *drbg,
-                      const unsigned char *adin, size_t adinlen);
+                      const unsigned char *adin, size_t adinlen,
```

Comment:
Is this a public API that you are changing here?

Reply:
No, I'm fixing the documentation for a public API.

Reply:
Please create a separate pull request for this fix, because it needs to be backported to 1.1.1

Reply:
See #8678